acceptance-test/src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaRebalanceAT.java

```diff
@@ -325,6 +326,25 @@ public void testCommitWhenDirectAssignment() throws Exception {
         assertThat(commitStatusCode, Matchers.is(HttpStatus.NO_CONTENT.value()));
     }
 
+    @Test(timeout = 10000)
+    public void testNoConflictOnTheSamePartitionRequest() throws Exception {
```

Comment:
that's unstable, because sometimes one stream takes partition, but the other one is too late, which results in 102 and 409
i appreciate ideas on test

Reply:
Maybe check that at least one of them will should have conflict?